lib/tasks/db.rake

```diff
@@ -45,8 +45,3 @@ end
 Rake::Task["db:structure:dump"].enhance do
   Rake::Task["db:structure:clean"].invoke
 end
-
-Rake::Task["db:seed"].enhance do
-  ENV["REFRESH_MATVIEWS_CONCURRENTLY"] = "false"
```

Comment:
`PreFillMonthlyScreeningReports` depends on Mat-views to be created before-hand. Hence, we had to move this to a direct function call inside `seed/runner.rb`